rinja_parser/src/lib.rs

```diff
@@ -626,7 +626,12 @@ fn path_or_identifier(i: &str) -> ParseResult<'_, PathOrIdentifier<'_>> {
             path.extend(rest);
             Ok((i, PathOrIdentifier::Path(path)))
         }
-        (None, name, []) if name.chars().next().map_or(true, char::is_lowercase) => {
+        (None, name, [])
+            if name
+                .chars()
+                .next()
+                .map_or(true, |c| c == '_' || c.is_lowercase()) =>
```

Comment:
I'm kinda confused if it is okay that this function returns `PathOrIdentifier::Identifier("_")` for the placeholder with this change. Before, `PathOrIdentifier::Path(vec!["_"])` would be returned. Actually, both look wrong to me.

Can you have a look what happens of you add a case ↓ before the changed one?

```rust
(None, "_", []) => fail.parse_next(initial_i),
```

Reply:
it didn't change anything as far as I could see.